.github/workflows/language_server_simulator_nballerina.yml

```
@@ -22,11 +22,11 @@ jobs:
         with:
           ref: ${{ matrix.branch }}
 
-      - name: Set up JDK 21
+      - name: Set up JDK 25
         uses: actions/setup-java@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)